helm-charts/splunk-otel-collector/templates/deployment-cluster-receiver.yaml

```diff
@@ -104,6 +104,31 @@ spec:
               mountPath: /splunk-messages
             - mountPath: /conf
               name: collector-configmap
+        - name: patch-log-dirs
```

Comment:
Nit: We might be outgrowing "log" functionality here for patch-log-dirs. If it works for now I'm fine with it. It might be time to refactor the names and images a little to make them more abstract to better address these new use cases.

Reply:
Yeah, something like patch-dirs would work for all configurations I guess?

Reply:
Yeah I'd go with patch-dirs.

Reply:
Does persistent buffering require the .Values.logsCollection.checkpointPath directory for checkpoints? I don't think it does. I recommend removing the code that sets up the logsCollection checkpoint directory for the cluster receiver and gateway to avoid potential race conditions. Since the agent is deployed to all nodes and the cluster receiver and gateway are only deployed to some, they may attempt to set permissions on the same directory simultaneously.

Reply:
To move forward with this
- I would just remove lines 111-113 for now so the agent and cluster receiver are not sharing on disk resources.
- Make sure to only include any of this new cluster receiver code if `.Values.splunkPlatform.sendingQueue.persistentQueue.enabled=true`